app/src/main/java/de/westnordost/streetcomplete/quests/address/AddAddressStreetForm.kt

```diff
@@ -0,0 +1,179 @@
+package de.westnordost.streetcomplete.quests.address
+
+import android.view.Menu
+import android.view.View
+import android.widget.EditText
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.widget.PopupMenu
+import de.westnordost.osmapi.map.data.LatLon
+import de.westnordost.streetcomplete.Injector
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.AbbreviationsByLocale
+import de.westnordost.streetcomplete.data.osm.elementgeometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.elementgeometry.ElementPointGeometry
+import de.westnordost.streetcomplete.data.osm.elementgeometry.ElementPolygonsGeometry
+import de.westnordost.streetcomplete.data.osm.elementgeometry.ElementPolylinesGeometry
+import de.westnordost.streetcomplete.quests.LastPickedValuesStore
+import de.westnordost.streetcomplete.quests.localized_name.AAddLocalizedNameForm
+import de.westnordost.streetcomplete.quests.OtherAnswer
+import de.westnordost.streetcomplete.quests.localized_name.AddLocalizedNameAdapter
+import de.westnordost.streetcomplete.quests.localized_name.LocalizedName
+import de.westnordost.streetcomplete.quests.localized_name.data.RoadNameSuggestionsDao
+import de.westnordost.streetcomplete.util.TextChangedWatcher
+import java.util.*
+import javax.inject.Inject
+
+class AddAddressStreetForm : AAddLocalizedNameForm<AddressStreetAnswer>() {
+    private var textField: EditText? = null
+    private var isPlaceName = false
+    private var defaultName = ""
+
+    override fun onClickOk(names: List<LocalizedName>) {
```

Comment:
functions should go below all variable definitions and constructor

Reply:
Fixed in https://github.com/westnordost/StreetComplete/pull/1782/commits/2522e90835c224c6d59a2feea053f266ed6dda89